src/frequenz/sdk/microgrid/component/_component.py

```diff
@@ -62,6 +95,7 @@ class Component:
 
     component_id: int
     category: ComponentCategory
+    type: ComponentType = InverterType.NONE
```

Comment:
It is missleading that Category type has InverterType as default.
Should we have any default at all?
Do you plan to add type for all components in this PR and then remove this default?
If yes, then please comment it for easier PR :)

Reply:
This is done.